client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -274,14 +276,18 @@ const SyncCardContainer = ( {
 
 	return (
 		<StagingSyncCardBody>
-			<SyncContainerTitle>{ translate( 'Data and File synchronization' ) }</SyncContainerTitle>
+			<SyncContainerTitle>{ translate( 'Database and file synchronization' ) }</SyncContainerTitle>
 
 			{ ! isSyncInProgress && (
 				<>
 					<SyncContainerContent>
-						{ translate(
-							'Sync your database and files between your staging and production environments—in either direction.'
-						) }
+						{ currentSiteType === 'production'
+							? translate(
+									'Pull changes from your staging site into production, or refresh staging with the current production data.'
+							  )
+							: translate(
+									'Refresh your staging site with the latest from production, or push changes in your staging site to production.'
```

Comment:
Would it be better to say ", or push changes from your staging site to production." as in the other message?

Reply:
Well, this message displays when you're in your staging site. I think the current version is the most correct.